manager/integration/tests/test_upgrade.py

```diff
@@ -162,23 +168,55 @@ def test_upgrade(longhorn_upgrade_type,
     host_id = get_self_host_id()
     pod_data_path = "/data/test"
 
-    pod_volume_name = generate_volume_name()
-
+    # Disable Auto Salvage Setting
     auto_salvage_setting = client.by_id_setting(SETTING_AUTO_SALVAGE)
-    setting = client.update(auto_salvage_setting, value="false")
-
+    setting = client.update(auto_salvage_setting, value="true")
     assert setting.name == SETTING_AUTO_SALVAGE
+    assert setting.value == "true"
+
+    # 2-1 Create volume0 with revision counter enabled
+    # attached to a node
+    disable_revision_counter_setting = \
+        client.by_id_setting(SETTING_DISABLE_REVISION_COUNTER)
+    setting = client.update(disable_revision_counter_setting, value="true")
+    assert setting.name == SETTING_DISABLE_REVISION_COUNTER
+    assert setting.value == "true"
+    volume0_name = 'lh-vol-0-test'
+    volume0 = create_and_check_volume(client,
+                                      volume_name=volume0_name,
+                                      size=SIZE)
+    volume0.attach(hostId=host_id)
+    volume0 = wait_for_volume_healthy(client, volume0_name)
+    volume0_data_before_sys_upgrade = write_volume_random_data(volume0)
```

Comment:
The volume creation logic looks duplicated for the three volumes. Suggest creating a function for it.